rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -153,107 +215,131 @@ components:
-            identifier is not guaranteed to be unique from Ion's perspective which might be
-            forced to use other properties to uniquely identify the system registering. This
-            identifier is typically generated differently based on the advertised product and/or
-            version.
+            identifier is not guaranteed to be unique from Ion's perspective which might be forced
+            to use other properties to uniquely identify the system registering. This identifier is
+            typically generated differently based on the advertised product and/or version.
           minLength: 1
           example: abcd-01234-xxxx
+        name:
+          type: string
+          description: >
+            The name of the remote system (a.k.a. site name). This name will be displayed to an
+            administrator when showing the remote system.
+          minLength: 1
+          example: Acme Intelligence Node
+        organization:
+          type: string
+          description: >
+            The organization the remote system belongs to.
+          example: Acme Organization
+        contact:
+          $ref: '#/components/schemas/ContactInfo'
+        product:
+          type: string
+          description: >
+            The type of product the remote system is. This will help Ion determine how to
+            communicate and what functionalities are available.
+          minLength: 1
+          example: ddf, alliance, ...
+        version:
+          type: string
+          description: >
+            The version of the product installed on the remote system. This will help Ion
+            determine how to best communicate with the system.
+          minLength: 1
+          example: 4.6.9
+        url:
+          type: string
+          format: uri
+          description: >
+            An externally accessible url where the remote system is hosting REST services that can
+            can be reached. For DDF-based systems, this would be the base url for the CSW and
+            GraphQL endpoints.
+          example: https://myhost:8993/services
     ResponseMessage:
-      description: Message information.
+      description: >
+        Response message which can optionally includes the echoed parameters that were understood
+        from the original request.
+      type: object
+      properties:
+        echoed.parameters:
+          $ref: '#/components/schemas/SystemInfo'
+    ErrorMessage:
+      description: Error message information.
       type: object
       required:
+        - status
         - message
       properties:
+        status:
+          type: integer
+          format: int32
+          description: The corresponding HTTP status code.
+          example: 200
+        code:
+          type: integer
+          format: int32
+          description: >
+            An optional more specific code corresponding to the error (specific codes need to be
```

Comment:
❓ _specific codes need to be documented_ : where?

Reply:
... along with the spec?